ui/selectors/transactions.js

```diff
@@ -102,18 +109,25 @@ export const smartTransactionsListSelector = (state) => {
   return state.metamask.smartTransactionsState?.smartTransactions?.[
     getCurrentChainId(state)
   ]
-    ?.filter((stx) => {
-      const isCancelledSmartTransaction = stx.status?.startsWith('cancelled');
+    ?.filter((smartTransaction) => {
```

Comment:
Quick note unrelated to this PR: Should we add a comment explaining that we only show swap smart transactions in the activity list and why?

Reply:
We also show pending non-swap smart transactions in the Activity list. I've added a few comments to make it a bit clearer.